scripts/resources/coredns-cm.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v1
 data:
   Corefile: |
+    supercluster.local {
```

Comment:
shouldn't this be supercluster.local:53 ???

Reply:
I added it to the config. If not added it will take 53 by default, but default config files define it explicitly.